terraform/files/postgres.yaml

```diff
@@ -0,0 +1,40 @@
+  1 init_config:
```

Comment:
Are these line numbers supposed to be here?

Reply:
argh - good catch!